integrationtests/api_helpers_test.go

```diff
@@ -32,7 +33,8 @@ func (a *apiHelper) logResult(desc string, err error) error {
 		return nil
 	}
 	addInfo := ""
-	if gse, ok := err.(ldapi.GenericOpenAPIError); ok {
+	var gse *ldapi.GenericOpenAPIError
+	if errors.As(err, &gse) {
```

Comment:
This little bug (the error is a pointer) was causing response bodies not to be logged.